packages/gov-app/libs/components/index.ts

```diff
@@ -0,0 +1,8 @@
+export { Button } from "@gov-app/libs/components/Button";
+export { Header } from "@gov-app/libs/components/Header";
+export { Layout } from "@gov-app/libs/components/Layout";
+export { ProposalAdvanced } from "@gov-app/libs/components/ProposalAdvanced";
+export { ProposalDetails } from "@gov-app/libs/components/ProposalDetails";
+export { Select } from "@gov-app/libs/components/Select";
+export { TextField } from "@gov-app/libs/components/TextField";
+export { WalletConnect } from "@gov-app/libs/components/WalletConnect";
```

Comment:
thanks for doing this, will have a hygen template to handle this to ease the pain :)